libvcx/src/api_lib/api_handle/issuer_credential.rs

```diff
@@ -188,6 +188,7 @@ pub async fn send_credential(handle: u32, connection_handle: u32) -> VcxResult<u
     Ok(error::SUCCESS.code_num)
 }
 
+#[deprecated(note = "Use combination of issuer_credential::revoke_credential_local and revocation_registry::publish_revocations instead")]
```

Comment:
This function also publishes locally revoked credentials, so it doesn't work as intended, and that is the reason for deprecation? But the suggested solution doesn't offer a different behavior...

Reply:
I see, this is addressed in #565.